libsolidity/Parser.cpp

```diff
@@ -824,18 +889,55 @@ pair<vector<ASTPointer<Expression>>, vector<ASTPointer<ASTString>>> Parser::pars
 	return ret;
 }
 
+int Parser::peekVariableDeclarationStatement() const
```

Comment:
I know it's only 3 distinct different return values but maybe for Future proofing return an `enum` here? Which would kind of self-document due to its naming what `1`, `-1` and `0` mean in the context of this function?